test/contrib/inference/dynamichmc.jl

```diff
@@ -1,15 +1,10 @@
-@stage_testset "dynamichmc" "dynamichmc.jl" begin
-    import DynamicHMC
+@testset "DynamicHMCExt" begin
+    using DynamicHMC
```

Comment:
```suggestion
    import DynamicHMC
```
should still be sufficient.